src/validation.cpp

```diff
@@ -63,162 +63,10 @@
 /**
  * Global state
```

Comment:
nit, in commit a8930a66b1 move-only: make the CChainState interface public


This comment wasn't accurate before the commit, nor is it accurate after the commit. And it made you move the code block below all the way down in the file.

Could instead remove this comment and keep the code block below in place? This also makes `git blame` and `git diff` work on the commit without workarounds.

Reply:
Yep, agree. Not sure how much it helps the diff, but that comment should be removed. I've pushed [the change](https://github.com/bitcoin/bitcoin/compare/64d0f872fcf6b61de9e4e62ce13dbbe3b00cbb6c..403e677c9ebbf9744733010e6b0c2d1b182ee850).